### PR TITLE
Null check

### PR DESCRIPTION
**Pull request by Thumimku**
## Purpose
In order to resolve https://github.com/wso2/identity-anonymization-tool/issues/123
Fixes https://github.com/wso2/identity-anonymization-tool/issues/123

### CODE REVIEW

File: components/org.wso2.carbon.privacy.forgetme.sql/src/main/java/org/wso2/carbon/privacy/forgetme/sql/module/DomainSeparatedSQLExecutionModule.java [hunk 1]
```diff
@@ -92,7 +92,8 @@ public void execute(UserSQLQuery userSQLQuery) throws ModuleException {
                 log.debug("Executed the sql query: {}.", userSQLQuery.getSqlQuery().toString());
             }
         } catch (SQLException e) {
-            throw new SQLModuleException(e);
+            //throw new SQLModuleException(e);
+            log.error(e.getMessage());
```
Comment (JKAUSHALYA) [inline components/org.wso2.carbon.privacy.forgetme.sql/src/main/java/org/wso2/carbon/privacy/forgetme/sql/module/DomainSeparatedSQLExecutionModule.java:96]: What is the reason not to throw the error?

**Comment by JKAUSHALYA** [comment]
Please add proper commit messages. 
Proper commit message ex: 
"Fix <what you fix>
Fix: <git-issue>."

Better to squash all the commits to a one.

File: components/org.wso2.carbon.privacy.forgetme.sql/src/main/java/org/wso2/carbon/privacy/forgetme/sql/module/DomainSeparatedSQLExecutionModule.java (hunk 1 above)
Comment (Thumimku) [reply components/org.wso2.carbon.privacy.forgetme.sql/src/main/java/org/wso2/carbon/privacy/forgetme/sql/module/DomainSeparatedSQLExecutionModule.java]: Provided logic is used to foresee masked errors. No needed to merge.

File: components/org.wso2.carbon.privacy.forgetme.sql/src/main/java/org/wso2/carbon/privacy/forgetme/sql/module/DomainSeparatedSQLExecutionModule.java [hunk 2]
```diff
@@ -48,7 +48,7 @@ public DomainSeparatedSQLExecutionModule(DataSourceConfig dataSourceConfig) thro
 
         try {
             dataSource = dataSourceConfig.getDatasource();
-            if (log.isDebugEnabled()) {
+            if ( dataSource != null && log.isDebugEnabled()) {
```
Comment (gayashanbc) [inline components/org.wso2.carbon.privacy.forgetme.sql/src/main/java/org/wso2/carbon/privacy/forgetme/sql/module/DomainSeparatedSQLExecutionModule.java:51]: Remove the white space in the beginning of the if condition

**Review by gayashanbc — APPROVED** [review]
Added a small comment

**Pull request merged**